### PR TITLE
Update_Deps_and_circleCI_job_test

### PR DESCRIPTION
**Pull request by TheCubicleJockey**
This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### CODE REVIEW

**Comment by TheCubicleJockey** [comment]
I have tested this in a Kepler cluster

**Comment by TheCubicleJockey** [comment]
Updated the dependencies and the storageclass api endpoint

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,20 +6,20 @@ orbs:
...
+      - image: cimg/go:1.19
     steps:
       - checkout
       - run: |
-          go get -u gotest.tools/gotestsum
-          mkdir -p /tmp/test-results/go
-          gotestsum --junitfile /tmp/test-results/go/results.xml -- ./pkg/... -cover -covermode atomic -coverpkg=./... -coverprofile=coverage.txt
+          curl -L "https://github.com/gotestyourself/gotestsum/releases/download/v1.8.1/gotestsum_1.8.1_linux_amd64.tar.gz" > /tmp/gotestsum.tar.gz
+          tar -xvf /tmp/gotestsum.tar.gz -C /tmp/
+          mv /tmp/gotestsum /usr/local/bin/
```
Comment (rbren) [inline .circleci/config.yml:22]: Looks like we lost the `gotestsum` command here?

```suggestion
          go install gotest.tools/gotestsum@latest
          mkdir -p /tmp/test-results/go
          gotestsum --junitfile /tmp/test-results/go/results.xml -- ./pkg/... -cover -covermode atomic -coverpkg=./... -coverprofile=coverage.txt

```

File: Dockerfile [hunk 2]
```diff
@@ -14,7 +14,7 @@ RUN go mod download
 COPY . .
 RUN go build -a -o gemini
 
-FROM alpine:3.15
+FROM alpine:3.16.2
```
Comment (rbren) [inline Dockerfile:17]: We like to unpin the patch version
```suggestion
FROM alpine:3.17
```

**Review by rbren — APPROVED** [review]

**Pull request merged**